### PR TITLE
Setting up app

### PR DESCRIPTION
**Pull request by beatadelura**
### TO DO:
* [X] Add pages controller
* [X] Create landing page
* [X] Create sign in page
* [X] Create login page
* [X] Link to login and sign in page from landing page

---

### TODO:

* [X] Board Management ( create, new , destroy, show )
* [X] Create action controllers and views
* [X] Create button Create Board. After click that button user is going to new page and have possibility to create board with adding board name
* [X] Add Save button on new board page which saves board in db and redirect to board page
* [X] Show all boards name list on homepage
* [X] User can view the board via clicking the name
* [X] Add back to boards list on show page

### CODE REVIEW

**Comment by beatadelura** [comment]
````
I think 'name' should be required field, so maybe try to add validation on presence in Model.
Do not forget about render 'new' when @board.save returns false. :)

```
Please check this out https://github.com/beatadelura/home_manager/commit/9b8234e74a19028c2aff65ed1a2677e8b03ba118

..but we do not show error messages.. anyway we don't know yet how messages will look like :)

**Comment by beatadelura** [comment]
```
When you finish this part ( create board ) please merge all changes to master.
We should using our flow with Pull request. It would be better.
```

I'm going to do that :)

File: app/views/boards/index.html.slim [hunk 1]
```diff
@@ -0,0 +1,10 @@
+h1 Boards
+= link_to "Log Out", destroy_user_session_path, method: :delete
+br
+= link_to "Create Board",  new_board_path
+
+.boards-list
+  ul
+  - @board.each do |board|
+    li
+      = link_to board.name, board_path(id: board.id)
```
Comment (PatrykTyszka) [inline app/views/boards/index.html.slim]: it's correct but you can do something like this:
board_path(board)

File: test/models/user_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```
Comment (PatrykTyszka) [inline test/models/user_test.rb]: after merge I propose to start writing some specs :)

**Comment by PatrykTyszka** [comment]
it looks legit ! :) :white_check_mark:

**Pull request merged**